packages/reason/reason.3.5.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "Jordan Walke <jordojw@gmail.com>"
+authors: [ "Jordan Walke <jordojw@gmail.com>" ]
+license: "MIT"
+homepage: "https://github.com/facebook/reason"
+doc: "http://reasonml.github.io/"
+bug-reports: "https://github.com/facebook/reason/issues"
+dev-repo: "git://github.com/facebook/reason.git"
+tags: [ "syntax" ]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml"         {>= "4.02" & < "4.09"}
+  "dune"          {>= "1.4"}
+  "ocamlfind"     {build}
```

Comment:
Are you sure ocamlfind is used anywhere? The only mentions I could find were related to esy and in `bspacks/reason_bspack.sh` but I'm not sure those two things are used here.

Reply:
Not sure, let me double check and get back to you.